CruCentralCoast/Extensions/UIColorExtensions.swift

```diff
@@ -9,6 +9,10 @@
 import UIKit
 
 public extension UIColor {
+    static var appTint: UIColor {
+        return .cruOrange
```

Comment:
bug

Reply:
Do you mean that Cru Orange as the app tint is a bug? 😉
I am saving the app tint fix for a different PR

Reply:
I will try that window?.tintColor trick in a different PR for app tint